src/celeritas/ext/GeantPhysicsOptions.hh

```diff
@@ -200,6 +212,30 @@ struct GeantPhysicsOptions
     //! Optical physics options
     GeantOpticalPhysicsOptions optical{
         GeantOpticalPhysicsOptions::deactivated()};
+
+    //! Initialize with no physics
```

Comment:
I think we've been meaning to move most of that list into a separate "EM physics" struct... wonder if now is the right time to do it?

Reply:
Let's defer.